contrib/rocksdb/template.py

```diff
@@ -0,0 +1,58 @@
+pkgname = "rocksdb"
+pkgver = "9.4.0"
+pkgrel = 0
+build_style = "makefile"
+make_cmd = "gmake"
+make_build_target = "static_lib"
+hostmakedepends = [
+    "bash",
+    "gmake",
+    "perl",
+    "pkgconf",
+]
+makedepends = [
+    "bzip2-devel",
+    "gflags-devel-static",
+    "linux-headers",
+    "lz4-devel",
+    "snappy-devel",
+    "zstd-devel",
+    "zlib-ng-compat-devel",
+]
+pkgdesc = "Embeddable, persistent key-value store for fast storage"
+maintainer = "Gnarwhal <git.aspect893@passmail.net>"
+license = "GPL-2.0-or-later OR Apache-2.0 OR custom:leveldb"
```

Comment:
```suggestion
license = "(GPL-2.0-or-later OR Apache-2.0) AND BSD-3-Clause"
```

the leveldb licence is just a normal bsd-3-clause